cli/command/service/helpers.go

```diff
@@ -25,7 +26,8 @@ func waitOnService(ctx context.Context, dockerCli command.Cli, serviceID string,
 		return <-errChan
 	}
 
-	err := jsonmessage.DisplayJSONMessagesToStream(pipeReader, dockerCli.Out(), nil)
```

Comment:
@martin-heralecky  I was going through the backlogs of pull requests, and saw this PR.

Looking at the existing code, the above is equivalent to;

```go
	err = jsonmessage.DisplayJSONMessagesStream(pipeReader,  dockerCli.Out(),  dockerCli.Out().FD(),  dockerCli.Out().IsTerminal(), nil)
```

(`DisplayJSONMessagesToStream()` is a wrapper for `DisplayJSONMessagesStream()`)

From the above; was there a reason to rewrite the code to use the terminal check here (instead of keeping the existing code, and only change `dockerCli.Out()` to `dockerCli.Err()` ?